internal/backend/cache/file.go

```diff
@@ -210,6 +210,9 @@ func (c *Cache) list(t restic.FileType) (restic.IDSet, error) {
 	dir := filepath.Join(c.path, cacheLayoutPaths[t])
 	err := filepath.Walk(dir, func(name string, fi os.FileInfo, err error) error {
 		if err != nil {
+			// ignore ErrNotExist to gracefully handle multiple processes clearing the cache
+			if(errors.Is(err, os.ErrNotExist)
```

Comment:
the brackets are somewhat messed up. It should be
```
if errors.Is(err, os.ErrNotExist) {
    return nil
}
```

Reply:
Oops, missed the closing brace.  Sorry about that.  Made the change directly on github as I don't have a dev environment for Go.  Would have benefited from some IDE syntax highlighting there. :)

Reply:
I've fixed the syntax error.  

Let me know if anything else is needed.